docs/src/pages/[platform]/connected-components/authenticator/intro.android.mdx

```diff
@@ -1 +1,9 @@
-The `Authenticator` component adds complete [authentication flows](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android) to your application with minimal boilerplate.
\ No newline at end of file
+import { Flex, Image } from '@aws-amplify/ui-react';
+
+The `Authenticator` component adds complete [authentication flows](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android) to your application with minimal boilerplate.
+
+<Flex direction="row" justifyContent="space-between">
+  <Image alt="Sign in with Amplify Android Authenticator" width="33%" src="/android/authenticator/sign-in.png" />
```

Comment:
Just to confirm, are we using a "33%" width for other screenshots throughout the docs?

Reply:
Offline conversation: there are no other instances of side-by-side images on the UI docs; 33% is too wide for mobile due to padding but 30% fits fine; narrow mobile form factors break the header dropdown as well